conf/pfmon.conf.defaults

```diff
@@ -717,7 +717,7 @@ timeout=10s
 # How long to keep a admin_api_audit_log entry before deleting it
 window=6M
 
-bandwidth_maintenance]
+[bandwidth_maintenance]
```

Comment:
I'm not involved in packetfence project but this kind of commit normally get squashed into previous ones.